setup.cfg

```diff
@@ -49,10 +49,10 @@ python_requires >= 3.9
 # For more information, check out https://semver.org/.
 install_requires =
     importlib-metadata; python_version<"3.8"
-    aiohttp
-    pandas
+    aiohttp<=3.9.1
```

Comment:
As a general rule, when you pin something, add an inline reason for it. Best pointing to a new Issue on this repo where updated circumstances can be shared and discussed.

Reply:
This comes from Victors work on the Tunes project.